prow/config.yaml

```diff
@@ -10529,6 +10529,26 @@ periodics:
       image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
       imagePullPolicy: Always
 
+- cron: '1 13 * * 2,4,6' # Run at 5:01PST (13:01 UTC) on even days
+  agent: kubernetes
+  name: ci-kubernetes-e2e-gce-scale-coredns
+  tags:
+  - "perfDashPrefix: gce-2kNodes-master"
+  - "perfDashJobType: performance"
+  labels:
+    preset-service-account: true
+    preset-k8s-ssh: true
+  spec:
+    containers:
+    - args:
+      - --timeout=540
```

Comment:
If you reduce the timeout for the job above, remember to also change this accordingly.